docs/GettingStarted.md

```diff
@@ -352,6 +353,7 @@ For a list of available integrations, and their configuration options, please re
 | MySQL2                   | `mysql2`                   | `>= 0.3.21`              | *gem not available*       | *[Link](#mysql2)*                   | *[Link](https://github.com/brianmario/mysql2)*                                 |
 | Net/HTTP                 | `http`                     | *(Any supported Ruby)*   | *(Any supported Ruby)*    | *[Link](#nethttp)*                  | *[Link](https://ruby-doc.org/stdlib-2.4.0/libdoc/net/http/rdoc/Net/HTTP.html)* |
 | Presto                   | `presto`                   | `>= 0.5.14`              | `>= 0.5.14`               | *[Link](#presto)*                   | *[Link](https://github.com/treasure-data/presto-client-ruby)*                  |
+| Que                      | `que`                      | `>= 2.2`                 | *gem not available*       | *[Link](#que)*                      | *[Link](https://github.com/que-rb/que)*                                        |
```

Comment:
`>= 2.2` should be updated to whatever version of `que` we'll end up supporting with this PR.
No need to update it now if you still don't know.

Reply:
for some reason, I thought it was the min ruby version required by the gem. Updated it to match. Thank you.

Reply:
Given your new tests pass, which exercise `que` and `ddtrace` together, I say we should state that we will be compatible with JRuby.

I found statement in the `que` repository: https://github.com/que-rb/que/issues/4#issuecomment-29561356. It reads to me that `que` it self doesn't have any issues with JRuby support, but other dependencies might. The issue I liked is also very old (~7 years old) and we have been able to use JRuby with a Postgres driver successfully for a while.

Reply:
Makes sense. Updated the jruby column to say `>= 1.0.0.beta2`. 

Thank you for the clarification.